var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -27,28 +27,31 @@
 
 class Paraview(Package):
```

Comment:
That's strange, how exactly is it crashing? I wonder if `spack build` tries to only run up to the build stage for every dependency.